src/template.html

```diff
@@ -426,6 +426,11 @@ <h3>{{topic.others.by-us-title}}</h3>
           summaryToggleButton.setAttribute('aria-expanded', 'false');
         }
 
+        setTimeout(() => {
+          themeToggleButton.classList.remove("hidden");
+          themeToggleButton.classList.add("transition");
+        }, 0);
```

Comment:
```suggestion
        // Workaround to prevent the toggle from doing a tiny animation
        // when the page loads. We can try to get rid of this workaround
        // some time in the future.
        setTimeout(() => {
          themeToggleButton.classList.remove("hidden");
          themeToggleButton.classList.add("transition");
        }, 0);
```